src/components/editor/Bindings/Selector.tsx

```diff
@@ -85,6 +90,15 @@ function Row({ collection, task, workflow, disabled }: RowProps) {
             } else {
                 setRestrictedDiscoveredCollections(collection);
             }
+
+            if (draftId) {
+                void deleteDraftSpecsByCatalogName(
+                    draftId,
+                    'collection',
+                    [collection],
+                    'remove'
+                );
+            }
```

Comment:
I am wondering if this is a safe call to ignore the outcome of. Would be a weird experience to tell a user "That delete didn't work - try again" but wondering what the risk is not handling different outcomes. Any thoughts?

Reply:
I don’t think it is important to flag these delete calls because a draft is concealed in the UI and the ultimate, “draft clean up” delete call triggered by the _Save and Publish_ CTA. The standard error handler (i.e., `callFailed()`) is used to catch that error and relay it to the user.

Once the continuous draft model is brought into the UI, the intermediate delete calls will require error handling.

Reply:
Sounds good